libpod/image/image_test.go

```diff
@@ -213,7 +210,7 @@ func TestImage_RepoDigests(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	for _, test := range []struct {
+	for _, tt := range []struct {
```

Comment:
OK, now I'm confused.  Lint wanted tt changed to test in the other code, but test to tt here?

Reply:
I wanted it to be consistent. `tt` as the loop variable, `test` for the variable we're using in the loop's body.